src/custom/state/price/updater.ts

```diff
@@ -158,15 +158,27 @@ export default function FeesUpdater(): null {
       const refetchPrice = !unsupportedToken && priceIsOld(quoteInfo)
 
       if (unsupportedNeedsCheck || refetchAll || refetchPrice) {
-        setLoadingQuote(true)
+        let quoteData: Pick<QuoteInformationObject, 'chainId' | 'sellToken'> | undefined = undefined
+        // did the trade parameters change? major update required
+        if (quoteInfo && !quoteUsingSameParameters(quoteParams, quoteInfo)) {
```

Comment:
check if quote refresh is due to param changes, read: HARD reload required. If it is a hard reload, pass dispatch the quote info to set it to null in state. Else just pass loading state (soft load)